src/Settings/settings.json

```diff
@@ -50,43 +50,72 @@
         "tip": "The default language used when opening new tabs"
     },
     {
-        "name": "Clang Format/Path",
-        "desc": "Path",
+        "name": "Clang Format/Program",
         "type": "QString",
         "default": "clang-format",
         "ui": "PathItem",
         "param": "PathItem::Executable",
         "tip": "The path to the Clang Format executable file",
         "old": [
-            "clang_format_binary"
+            "clang_format_binary",
+            "clang_format/path"
         ]
     },
+    {
+        "name": "Clang Format/Arguments",
+        "type": "QString",
+        "tip": "The arguments passed to clang-format. It should NOT contain \"-i\"."
+    },
     {
         "name": "Clang Format/Style",
-        "desc": "Style",
         "type": "QString",
         "default": "BasedOnStyle: Google",
         "ui": "QPlainTextEdit",
-        "tip": "The Clang Format style options, which are often saved in a .clang-format configuration file.\nYou can learn about it at <https://clang.llvm.org/docs/ClangFormatStyleOptions.html>.",
+        "tip": "The Clang Format style options, which are usually saved in a .clang-format configuration file.\nYou can learn about it at <https://clang.llvm.org/docs/ClangFormatStyleOptions.html>.",
         "old": [
             "clang_format_style"
```

Comment:
Did you forgot to add old key for clang format style?

Reply:
The key wasn't changed.